.circleci/config.yml

```diff
@@ -220,67 +253,128 @@ jobs:
       image: "18.2.0"
     <<: *test_template
 
+  unit_tests_node20:
+    executor:
+      name: node
+      image: "20.19"
+    <<: *test_template
+
+  unit_tests_node22:
+    executor:
+      name: node
+      image: "22.18"
+    <<: *test_template
+
   integration_tests_gatsby_source_wordpress:
```

Comment:
don't know why this one integration test has a whole custom setup. I didn't look into it. seems to work.